python/GangaTest/__init__.py

```diff
@@ -5,6 +5,8 @@ def getEnvironment(c = None):
     return {}
 
 def loadPlugins( config = None ):
+    import Lib.GListApp
```

Comment:
In that case I think a separate PR isolating the current tests again from the GangaTest is needed, there is some bleed already happening between these 2 codebases and I'd rather not end up fixing one at the expense of the other until we have the majority of all test in the new test system.